### PR TITLE
Update APIs for "Aws filtered by Ocp" overview

### PR DESCRIPTION
**Pull request by dlabrecq**
Updated the APIs for the "Aws filtered by Ocp" overview page

https://github.com/project-koku/koku-ui/issues/1340

### CODE REVIEW

File: src/api/reports/report.ts [hunk 1]
```diff
@@ -82,7 +82,10 @@ export const enum ReportType {
 
 export const enum ReportPathsType {
   aws = 'aws',
+  awsCloud = 'aws_cloud',
   azure = 'azure',
-  ocpCloud = 'ocp_cloud',
+  // azureCloud = 'azure_cloud',
```
Comment (boaz0) [inline src/api/reports/report.ts]: should we keep this comment?

File: src/api/reports/report.ts [hunk 2]
```diff
@@ -82,7 +82,10 @@ export const enum ReportType {
...
   ocp = 'ocp',
+  ocpCloud = 'ocp_cloud',
+  // ocpUsage = 'ocp_usage',
```
Comment (boaz0) [inline src/api/reports/report.ts]: should we keep this comment?

File: src/api/reports/reportUtils.ts [hunk 3]
```diff
@@ -14,15 +17,24 @@ export function runReport(
...
+      break;
     case ReportPathsType.azure:
       report = runAzureReport(reportType, query);
       break;
-    case ReportPathsType.ocpCloud:
-      report = runOcpCloudReport(reportType, query);
-      break;
+    // case ReportPathsType.azureCloud:
+    //   report = runAzureCloudReport(reportType, query);
+    //   break;
```
Comment (boaz0) [inline src/api/reports/reportUtils.ts]: should we keep this comment?

File: src/api/reports/reportUtils.ts [hunk 4]
```diff
@@ -14,15 +17,24 @@ export function runReport(
...
     case ReportPathsType.ocp:
       report = runOcpReport(reportType, query);
       break;
+    case ReportPathsType.ocpCloud:
+      report = runOcpCloudReport(reportType, query);
+      break;
+    // case ReportPathsType.ocpUsage:
+    //   report = runOcpUsageReport(reportType, query);
+    //   break;
```
Comment (boaz0) [inline src/api/reports/reportUtils.ts]: should we keep this comment?

Comment (dlabrecq) [reply src/api/reports/reportUtils.ts]: Merged the comments via a separate PR for the OpenShift Usage overview

**Pull request merged**